static/app/views/issueDetails/streamline/foldSection.tsx

```diff
@@ -79,6 +81,41 @@ function useOptionalLocalStorageState(
     : [persistedState, setPersistedState];
 }
 
+function useScrollToSection(
```

Comment:
Guess we'll need some generic hooks in core eventually! This one seems too coupled to issue details at the moment, but still better to extract into a local hook! 👍